pages/glossary/embedded-image.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Embedded Image
+key: embedded-image
+unambiguous: true
+objective: true
+input_aspects:
+  - DOM tree
+---
+
+An element presents an _embedded image_ when any of the following is true:
+
+- the element is an `img` element with a non-empty [source set][]; or
+- the element is an `input` element with a `type` [attribute value][] of `image` and its `src` [attribute value][] is not empty; or
+- the element is a `canvas` element with an [image][canvasImage] [drawn][drawImage]; or
+- the element is an `object` element with a `data` [attribute value][] referencing a resource with an [image MIME type](https://mimesniff.spec.whatwg.org/#image-mime-type); or
+- the element is an `svg` element having one or more `image` [descendants][] with a non-empty `href` [attribute value][]; or
+- the element has a [computed][] [`background-image`][background-image] CSS property with at least one [`image`][css-image] that is a [url reference][url-reference].
```

Comment:
Similar to the above, background can have multiple images. I think the rule needs to handle that.

Reply:
The first assumption was added exactly because of such cases. Given that we are testing every rendered image source, some may overlap others, therefore we check each individually and we might fail some instances where text in one source is in fact obscured by other sources.

Reply:
But you don't though. The applicability isn't testing visibly rendered image resources. It's testing elements, which may render multiple images at the same time.